src/ArrayOfSets.hpp

```diff
@@ -115,10 +115,16 @@ class ArrayOfSets : protected ArrayOfSetsView< T, INDEX_TYPE >
 
   /**
    * @brief Default move assignment operator, performs a shallow copy.
+   * @param src The ArrayOfSets to be moved from.
    * @return *this.
    */
   inline
-  ArrayOfSets & operator=( ArrayOfSets && ) = default;
+  ArrayOfSets & operator=( ArrayOfSets && src )
+  {
```

Comment:
Memory leak.

Reply:
When we setup CI  for LvArray, we should have a valgrind test.